compiler/src/iree/compiler/Modules/IO/Parameters/Transforms/BUILD.bazel

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("//build_tools/bazel:build_defs.oss.bzl", "iree_compiler_cc_library", "iree_gentbl_cc_library")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+iree_compiler_cc_library(
+    name = "Transforms",
+    srcs = [
+        "ParameterizeGlobals.cpp",
+        "PassDetail.h",
+        "Passes.cpp",
+    ],
+    hdrs = [
+        "Passes.h",
+        "Passes.h.inc",
+    ],
+    deps = [
+        ":PassesIncGen",
+        "//compiler/src/iree/compiler/Dialect/Stream/IR",
+        "//compiler/src/iree/compiler/Dialect/Util/IR",
+        "//runtime/src/iree/base",
+        "//runtime/src/iree/base/internal:file_io",
```

Comment:
sketchy internal reference - we want to use LLVM utils to map the file instead - FileOutputBuffer should do it and handles other things we want too (writes to a temp file and moves it into place only on success, otherwise deleting it if we fail to write, etc). you can use getBufferStart() / getBufferSize() to pass to iree_io_file_handle_wrap_host_allocation and then have the release callback there be a no-op if you manage the unique ptr yourself (you need to commit() at the end to finish the write)